### ISSUE TITLE
Docker version is still 0.4.0

### ISSUE THREAD

**Issue by nanosonde** [spacemanspiff2007/HABApp#33]
Hi!
I had deleted the container and image and reinstalled from tag "latest" which should be:
https://github.com/spacemanspiff2007/HABApp/blob/d9e25cc9fe5ba4c56c809f6fb62bb52c4403736d/HABApp/__version__.py#L1

However, the log file still shows version 0.4.0.

**Comment by spacemanspiff2007** [comment]
Sometimes it takes time for PyPi to serve the most recent version.
Maybe try again in an hour or so.

**Comment by nanosonde** [comment]
Ok.
But can the Dockerfile be modified in a way that it uses the recent GIT version immediately?
I find it quite disturbing that a fetched docker image does not match the latest GIT master.

**Comment by nanosonde** [comment]
Maybe a solution would be to do it like https://github.com/home-assistant/appdaemon

**Comment by nanosonde** [comment]
Doing it the appdaemon way would also provide the benefit of being able to install other python requirements during container startup. It checks for a "requirements,txt" file at every startup in the config folder and install the modules.

**Comment by nanosonde** [comment]
Your Dockerfile would just need this if I am not wrong:

```
RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app
COPY . .

# Install
RUN pip3 install .
```
instead of just
```
WORKDIR /usr/src/app
RUN pip3 install HABApp
```

**Comment by spacemanspiff2007** [comment]
I modified the file as suggested. Can you verfify it is working correctly, please?

**Comment by nanosonde** [comment]
After removing the old image and the old container and reinstalling the image. The newly created container from the "latest" tag starts correctly and shows 0.4.6.

I would like to wait until you provide another release to see if upgrading the image of an exsting container works.

**Comment by spacemanspiff2007** [comment]
I pushed 0.4.7 - have you already tried upgrading?

**Comment by nanosonde** [comment]
Yes, it worked without any problem.
On the Synology NAS I had just selected "Download" with tag "latest" to download the new image.
It will overwrite the old one on the NAS.
Then I have stopped the container. After selecting "Action" -> "Clear" on the HABapp container, the container was still there, but without any content.
After start the container again, everything immediately worked as it should.
So it is very easy to upgrade the HABapp container.
